### PR TITLE
PW-9: Adding local and Jenkins sonarqube capabilities and also ....

### PR DESCRIPTION
**Pull request by lenardw1**
...capabilities to run unit and integration tests locally and in Jenkins. A task to check for minimum project code coverage was also added.

### CODE REVIEW

File: devops-accounts-service/settings.gradle [hunk 1]
```diff
@@ -1,18 +0,0 @@
...
- */
-
-/*
-// To declare projects as part of a multi-project build use the 'include' method
-include 'shared'
-include 'api'
-include 'services:webservice'
-*/
-
-rootProject.name = 'devops-accounts-service'
```
Comment (RussianPrussian) [inline devops-accounts-service/settings.gradle]: When we remove this file, what happens when we just want to build this project?

Comment (RussianPrussian) [reply devops-accounts-service/settings.gradle]: Okay. no problems there.

**Comment by lenardw1** [comment]
Yes sourcesets is just for integration tests. Gradle already knows where the unit test sources are.

**Comment by RussianPrussian** [comment]
great.

**Pull request merged**